static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js

```diff
@@ -53,7 +53,7 @@ define([
                 siteMessageLinkName: 'adblock message variant ' + message.id,
                 siteMessageCloseBtn: 'hide'
             }).show(template(messageTemplate, {
-                supporterLink: adblockLink + '?INTCMP=adb-mv-' + message.id,
+                linkHref: adblockLink + '?INTCMP=adb-mv-' + message.id,
```

Comment:
@Calanthe this is the fix that allows the banner to display again... as @joelochlann explained to me, the client-side template-rendering for the banner will fail completely if it doesn't have the correct `linkHref` variable available - @davidrapson [changed the variable name in the template](https://github.com/guardian/frontend/pull/10867/files#r42518034) with PR 10867